### DISCUSSION TITLE
MeshCentral news, Sponsorship and Opportunities

### DISCUSSION THREAD

**Discussion by Ylianst** [Ylianst/MeshCentral#4795] [Announcements]
As some of you know, Bryan and I work at Intel and have both done so for over 20 years. Intel is wonderful and has great remote management technology (Intel Active Management Technology, Intel AMT) and as part our work at Intel we built many open-source tools that help people remotely manage their computers.

Intel has been going thru harder times and cutbacks of late and during the first week of November (2 weeks back) both Bryan and I have been downsized from the company. So, we are both going to be looking for other work. This came as quite a surprise to us as MeshCentral, MeshCommander, MeshCMD and other tools where widely used and beloved by many. They promoted Intel technologies, in my view, quite effectively. In any case, this does mean a life change and the stress and uncertainty that comes with it. I have been working on some version of MeshCentral for over 12 years now and the open-source community around MeshCentral is absolutely wonderful and I want to keep it going.

MeshCentral is a community project and obviously, it’s going to stay on GitHub and NPM and I will continue to maintain it at a slower rate since I have to figure out my next steps. For example, in the next week Bryan and I will get a new MeshAgent tested for release along with more fixes, latest OpenSSL version and more. Longer term, how much time we spend on it depends where we land.

I want to be super open with the community, please don’t hesitate to ask questions below. We are in the Portland, Oregon area and are looking for a good fit. Got plenty of mad design/coding/video skills that can be put to good use. My [LinkedIn page is here](https://www.linkedin.com/in/ylianst/), if are a fan of MeshCentral or just want to say hi, please feel free to connect.

**Comment by PathfinderNetworks** [comment]
I'm sorry to hear that, Ylian.  20 years is a long time to be with a company and to have something like this happen.  But things are getting crazy (or rather crazier) and no one is immune.  I lost my corporate IT management job back in 2004 and have been a self employed business owner ever since (I was actually a business owner since 1997 but didn't go full time with the IT business until I was let go from that corporate job).  It's even more important in this environment to be in charge of our own destiny.  I can never see myself working for someone else ever again.  Have you and/or Bryan considered going out on your own?

**Reply by Ylianst** [reply]
Hi. I do want to note that you are one of the few around here to have deployed MeshCentral v1 a long time ago. I like to think we go way back. Starting a business is not something I ever did or have any knowledge about. I do want to look at what is out there first, but there is not a week that goes by without someone suggesting I do exactly that. I rarely come across this suggestion coming from someone who did it themselves, so, coming from you, that carries more weight.

**Reply by PathfinderNetworks** [reply]
MeshCentral, starting way back with your first version, has been the most important tool for running my business.  It was a total game changer way back then.  
I will say, in regards to starting a business, I have known many people that decided to go out on their own and start a business only to have it fail months after they start.  So it's definitely not for everyone.  Many of these people were very highly skilled and know their area of expertise extremely well.  But they didn't know or understand what it takes to run a business.  
So just keep that in mind.  There is a lot to running a successful business, most of which has nothing to do with the product or service that is being provided.  I spend more time on the back end of the business than I do providing the IT services and products my company offers  
But, if you can make it work, it is VERY rewarding and gives you a greater amount of freedom than you can ever get from working for someone else. 

All that said, I have always thought there has to be some way MeshCentral could be monetized or offered as a commercial product while still offering the opensource, free option you do now.  If it is something you decide to pursue I would love to be involved in some way.  I am looking to retire from my business at some point soon (I was going to sell out and retire this year but then the stock market tanked and so did my holdings).  I'm not a coder in any way, shape, or form though.

**Reply by vithurshanselvarajah** [reply]
I have to agree on this. Meshcentral is a tool lots of businesses would like to be able to pay for. I use Meshcentral for my own business as well as my day job and would see no issue signing up to a enterprise subscription model. 

As long as you price fairly and dont go money crazy (like logmein and a few other services) you could be a platform changing service. 

A free tier and a enterprise level tier would suit really well. Depending on local laws you could start off as a sole trader and take it from there. Its what im doing with my business. Once it grows big enough ill consider a limited business but for the moment sole trader means less paperwork for taxes. If you want to reduce running costs in the interim let me know. I provide hosted services. Considering how long ive been able to use Meshcentral for free the least i can do is provide you with some resources for free!

**Reply by renueman** [reply]
Hello,
I just want to add that I was in I.T. up until 2008 when the economy crashed. I worked on high speed printers, servers and networks all on contract. Got laid off. I started my own business. I have to say that things were tough in the beginning. First, I could not even afford a McDonalds coffee. Thats how bad it was. However, I soon found my nitch and have been steadily growing ever since. Today I am a lot happier and stress free. I do not earn as much money as I did working for a large company but the trade off is well worth it for me. 

Ylianst, I hope things work out for you as I am sure they will. If you want to start asking for donations to support Meshcentral then I have no problems with that. In fact, I think you should now.

**Comment by si458** [comment]
So sorry to hear! A few ideas for you...

1. Have you thought about making a meshcentral organisation on github then adding admins/mods to help with issues and pull requests?

2. Have you through about like the portainer model where it remains opensojrce but then sell subscriptions to organisations for support/feature requests?

3. Have you asking Intel about number 2 but under their name/brand?

**Reply by Ylianst** [reply]
Good suggestions. For number 1, I have not looked at this. I am plenty open to more contributions and just have to take care to keep security of the code a top priority.

For number 2, if I start something about MeshCentral that will certainly be a part of it. I have gotten plenty of requests for a paid support model.

For number 3, Intel and other large organizations have a lot of processes you have to go thru in order to put a brand on software. This is understandable as you don't want to release software under your brand that is not secure or not high quality. So, it's possible but in general its more costly. Some smaller companies already do this and have lots of success with this model.

**Reply by jsastriawan** [reply]
I don't want to express my feelings about all that has happened, you knew it already. I'll do what I can to help. 

Please consider setting up an LLC so that you can get a contract work. I am sure many people can help advising about this part.

Your mentorship is life changing and your coding efficiency is totally an eye opener to me. Some people say maintaining your code is hard, but having enough understanding of your flow and how you craft your code make it easier for me to work with your code.

Because you are so nice and kind to the world, I truly believe the world will be nice and kind to you.

**Comment by Fidreus** [comment]
Make business of meshcentral, 

Examples:
For many corporations it's bad that they can't buy a support contract for a program. It makes them feel more safe. 

Break out new enterprise functions from the source code and sell that.

Sell the cloud meshcentral as a SaaS.
etc

Once you go with your own business the only thing you will regret is that you didn't do it sooner (I have done the same journey).

**Reply by Ylianst** [reply]
Thanks for the suggestions. Yes, if I start something around MeshCentral, this is going to be the things I do. I have never started by own business and you may be right that it would be interesting. The trade-off is that I can generally do well in large corporations, so I will look into both areas.

**Comment by petervanv** [comment]
So sorry to hear that ylianst and brian, They are dumb too fire you.

Keep up the good work and hope for you the sort things out quick.

**Reply by Ylianst** [reply]
Thanks you. I do want to mention that I absolutely loved working for Intel and would certainly return in the future if the opportunity was given. Due to recent financials, I completely understand the difficult position teams are being put under.

**Comment by lfabry** [comment]
Hi
sorry to hear that. 
Please keep up the good work.
Was defnitely wondering how we could support  your great work.

**Reply by Ylianst** [reply]
Thanks! Being part of this community is already wonderful support.

**Comment by TerAnYu** [comment]
о-О
Sympathize with you.
I hope that you can find a more profitable and / or interesting place to work.
I wish you succeed!
Please do not be discouraged, although it is difficult to do, you will succeed!

**Reply by Ylianst** [reply]
Thanks. No worries, we are doing well and have good skills. Just a question of finding a great fit.

**Comment by The-Real-Admin-Guy** [comment]
I feel for you!
One Door closed many may now have an opportunity to open!
I must say, with your Intel work experience + your excellent coding background, combined with the educational videos you created for Meshcentral over the years and the total endorsement from the MeshCentral user base you  probably have a lot of amazing opportunities ahead!

In the mean time I would consider freelance with something like **Fiverr**   
https://business.fiverr.com/business?
https://www.fiverr.com/start_selling

Here is another plug to help you get started promoting yourself in the business community:
https://www.bni.com/

Becoming a business appears easy, but trust me its not. It's truly unpredictable and hard work doesn't always pay. 
You need to upgrade on a lot of skills and competition is brutal, and right now the economy is in shambles.  

However the rewards for those who succeed can be significant.  Considering what I have witnessed,  I think you may have a fighting chance at success...

I wish you both the very best!

**Comment by JSuenram** [comment]
I am sorry that you loose your jobs. I have a small "dumb" idea without having any connection to MESH or TRMM.
But what about sticking together with the guys from Tactical RMM and make something really cool together? Just my 2 cents.

**Reply by Ylianst** [reply]
Thanks for the note. Not a dumb idea at all. I will be looking at the market place in the next weeks and months and see what is available.

**Comment by SomeGuru** [comment]
Ylian, Bryan,

Sorry to hear the news, shock is the feeling right now... Since Feb 2019 until now you guys have come with innovations in MeshCentral to make corporate easier to allow running of such a great server. I totally agree with your stance that right now the time is to focus and narrow the gap for several corporations. We cannot hold resentment, but gratitude for the great number of years Intel supported your endeavors for MeshCentral 1 & 2. You were dedicated to the task at hand and your Job. So I commend you on a "Job well done!". I can only hope and pray that someone will pick you up and allow you to continue on with MeshCentral and maybe even give you even more resources to achieve even great things. Remember your best days are still ahead of you! 

Kindest regards,
SomeGuru

**Reply by Ylianst** [reply]
Thanks. Yes, working at Intel has been a blast and we never know, we may be back in the future. It's certainly been a shock for us. Thank you so much for your kind words.

**Comment by Ylianst** [comment]
One thing I do want to note is that without being at Intel, it's going to be nearly impossible to continue to support Intel AMT going forward. Building software for Intel AMT that works well requires a hardware rich environment with access to lots of computers (different versions, SKU's, etc). I will no longer have access to latest hardware and so, I am thinking I may put AMT support behind a config.json switch but disable it by default. This way, it lowers expectations. Any input on this is appreciated.

**Reply by dinger1986** [reply]
I think thats a great idea, I would be surprised if the majority of users use AMT anyway, maybe wrong though

**Reply by thermionic** [reply]
@dinger1986 AMT support was why I was interested in MeshCentral.

**Reply by JSuenram** [reply]
We use Intel AMT regularly, but mostly to "Start/Wake PCs on remote sites". When there is no other PC to Wake on LAN this is the only way. Would be very sad if that feature is going away.

If I get a guess about the costs, my company as a last resort would be ready to takeover MeshCentral-Project to continue development. But I also guess this will slow down thinks a lot... However, at last we could provide paid enterprise support from the beginning as we are heavily using MeshCentral and also had made some "very" small development steps inside Mesh-Code...

**Reply by silversword411** [reply]
Is there any other official Intel software to manage and use amt? 

If there isn't, Intel must be abandoning the entire AMT idea

**Reply by dxdemetriou** [reply]
In our case, the main role of the AMT is the remote powerbutton press and then the boot method. Some tests could be possible to be done in computer labs (and I would like to help whether is needed), but sure here will be more rare for the latest hardware as our currently latest CPUs are Intel Core i5-8500, so probably most of the times will be for bugs and fixes. Here we started to use MeshCentral, and because of MeshCentral we started to use AMT. I hope to be able to be maintained at least the WOL functionality of AMT (if Intel is not having different plans for their future).

**Comment by asasin114** [comment]
I'm sorry to hear you and Brian have been let go. I don't think Intel truly realizes what they've let go and I hope they bring you both back. In the meantime I wish you success in this part of your journey.

Have you talked with @wh1te909, @silversword411 and @dinger1986 over at @amidaware? TacticalRMM is an amazing tool and the fact that the integrated MeshCentral out of the gate is huge! I've been blown away with the tools they delivered and the product that you've built over the past 12 years and I'm so thankful for the hard work you've put in. I'd be over the moon to see you all working closely to build the most bulletproof remote access and remote management and monitoring tool around!

**Comment by bradyrf** [comment]
Ylian,
Sorry to hear you and Intel are parting ways. Your resume and proven accomplishments I'm sure will be very attractive to a perspective employer. Having that said, I think you could probably open a company and be very successful. Have you considered offering "Premium Features" add ons for Mesh Central? I have a feature request in for a device idle status, I would be happy to pay for. Reach out to my Gmail if your interested. I really need this to completely move away from ConnectWise and would much rather pay you than keep paying maintenance on CW. Like others have suggested, maybe come up with a "Commercial Edition" where the "Premium features" are added at an accelerated rate. Whatever you decide, I'm certain, given your talents and personality, you will be successful. Thank you for all of the amazing work you do for the community.

**Comment by PrplHaz4** [comment]
The quality and value you deliver as a truly open and transparent solution without vendor lock-in is unmatched.  Clearly many people would not be able to run their businesses (at the same margins and risk tolerance) without you.

You owe it to yourself to start a Patreon, or some other low-effort funding model to allow the (many) IT shops that couldn't be run without this product to contribute to it's longevity.  It would likely not generate an Intel salary, but gives you and Brian some incentive to stay involved, and potentially allows your users some assurance that the product is not going away.

Wishing you guys the best of luck either way. You don't owe us anything, have already provided a great deal of value to the community, and have always been a great example of how open source should work - so thanks for everything thus far!

**Comment by 12nick12** [comment]
I'm sorry to hear about that. You guys should at least enable sponsorships or donations someway. I would gladly donate some money every month. It won't be a bunch, but I pretty sure many other people would as well.

**Comment by NiceGuyIT** [comment]
Hey @Ylianst and @krayon007, it should be clear from the responses above that the community is behind you whatever you choose. I agree with those who stated that starting your own business is fruitful in the long term but hard to get started and manage in the short term. It's easy to take care of the technical side of the business and it's much harder to take care of the business and customer facing side of things. I also agree with those who suggest some sort of financial compensation. Not everybody has the programming knowledge to contribute code, yet they want to contribute to a project they use.

Why not create a MeshCentral organization in GitHub and move the repos to the organization? You and Brian can be the owners and add members as you see fit (after they've proven themselves to be a valuable member of the community.) You can setup sponsorship for those who want to give back financially. Setting up bug bounties for issues allows people to fund issues that are important to them.

The above can be done without creating a business entity and at the same time provide a means for the community to give back. You and Bryan are in control, guiding the direction of the project.

**Reply by JSuenram** [reply]
> Hey @Ylianst and @krayon007, it should be clear from the responses above that the community is behind you whatever you choose. I agree with those who stated that starting your own business is fruitful in the long term but hard to get started and manage in the short term. It's easy to take care of the technical side of the business and it's much harder to take care of the business and customer facing side of things. I also agree with those who suggest some sort of financial compensation. Not everybody has the programming knowledge to contribute code, yet they want to contribute to a project they use.
> 
> Why not create a MeshCentral organization in GitHub and move the repos to the organization? You and Brian can be the owners and add members as you see fit (after they've proven themselves to be a valuable member of the community.) You can setup sponsorship for those who want to give back financially. Setting up bug bounties for issues allows people to fund issues that are important to them.
> 
> The above can be done without creating a business entity and at the same time provide a means for the community to give back. You and Bryan are in control, guiding the direction of the project.

Fantastic Idea in my opionion!

**Reply by Ylianst** [reply]
Agreed. I am looking at various options.

**Comment by JSuenram** [comment]
In german we have a wording for this:

butter by the fishes! Who will donate at least 50$ per month if possible? Thumbs up for yes!

**Reply by 12nick12** [reply]
> In german we have a wording for this:
> 
> butter by the fishes! Who will donate at least 50$ per month if possible? Thumbs up for yes!

I couldn't do $50 unfortunately, but could do $25.

**Reply by JSuenram** [reply]
Which makes 75$ already every month.... stay positive! ;-)

**Reply by vithurshanselvarajah** [reply]
I'd happily do $25 as well. For each instance. So thats 3

**Reply by Ylianst** [reply]
In general, I do think MeshCentral should remain mostly free with sponsors and a way for people who need it to get a support contract. This said, it all depends on where I land.

**Reply by cfoellmann** [reply]
@Ylianst I think your just getting your footing at Microsoft.
Do you have any idea how proceed with meshcentral yet?
Our company might move over to meshcentral and there will be a budget freeing up.
I imagine (hope) that a definite call for sponsorship (like $0.5/endpoint/month) might get you some funds BUT I know that money cannot buy you time in the long run. So you might need to look into outsourcing work with that money!?

**Comment by The-Real-Admin-Guy** [comment]
Just my two cents again,
Asking too much $$$ will simply results in being ignored by the great many....

I was doing some thinking, and I realised that as an artist,  (I'm a guitarist),  many of us monetise via support clubs instead of selling individual pieces or even albums where we often let the fan base pay directly as they feel comfortable with.   Providing quality usually results in superior long term support for creators and a greater residual than direct selling as such...

Maybe considering something like Patreon?
https://www.patreon.com/home

I mean you already have a tech fan club, just let it formulate into a support club!
Simply add the option to your bios and other similar endeavours, and you never know?

Just a thought!

**Reply by Ylianst** [reply]
Before asking for any money I want to figure out where I will land first. I don't think donations would even come close to income of a corporate job and I don't want to ask for money if I am not maintaining MeshCentral for some significant time each week. However, this is certainly an option. If I where to retire from corporate America, I would probably go in this direction.

**Reply by SomeGuru** [reply]
Well, that is not an option considering that you are only 33 Years old now... you have lots of life still in front of you Ylian... Unless there is a large lump some lottery winning, I don't think retirement this early from corporate america will do well on the financials... 

SomeGuru

**Comment by K-O-K** [comment]
Hi @Ylianst and @krayon007,

First let me tell you: Thanks for all the work and effort you guys have put it to this amazing project so far.

I was reading all this and one of the first things I'd do is take on the @vithusel offer to start what @Fidreus suggested, selling the cloud Meshcentral as a SaaS subscription model, that way you could monetize a bit and start from there; you could take advantage of the {domain} part of the config.json file and share the server for a few clients.. offering the possibility to personalize the installation.

I'm not an expert but I offer you my help with my Infrastructure skills, I recently connected with you over LinkedIn and over Gmail several months ago, so feel free to reach out. It's the least I can do after you what you created did for me.

SN

**Comment by ghost** [comment]
Hi, Ylianst and Krayon007..

There's already been said a lot about what happend and also about the future which we think is something to look forward to because it also means new opportunities.

We have been able to rely on you both quite some time - already since MeshCentral 1.0 and offcourse a big thanks for the support and effort.  Others mentioned already some support like donations, commercial licensing.. but this is something that also needs commitment. Looking at the future we're more then willing to help in any way and hopefully MeshCentral 2.0 wil be continually supported.

Don't underestimated what is, MeshCentral 2.0 is a product which is capable of things not many others can offer.

Kind regards,

Videomanager Support

**Comment by lfabry** [comment]
I completely agree with @VmSupport and everyone, Meshcentral 2.0  is a great solution with a lot of potential.
Would be very pleased to help to get this moving to a bright future.

**Comment by unibogdv** [comment]
Intel did some not very brite moves about their vPro CPUs:
1. They disabled the provision of AMT though USB
2. They created a notebook HW platform without the possibility to use Thunderbolt network adapters for AMT
3. They ignored the immense amount of work you and Bryan did and what a great SW is MeshCentral

**Comment by sharifm-informatica** [comment]
@Ylianst ,
I have some ideas worth a discussion on meshcentral managed and SaaS services. I have some infrastructure for deployment of meshcentral on Kubernetes. Would love a chat if you are open to the idea.

**Comment by EDIflyer** [comment]
I was glad to read that Microsoft snapped you up, @Ylianst!! Intel's loss is their gain.

Hopefully they might offer something like Google along the lines of '20% time' for developers to work on their own projects ;) (I can but dream!)

**Reply by sharifm-informatica** [reply]
Congratulations @Ylianst I wish you all the best in your new position. I am very happy for you.

**Comment by PathfinderNetworks** [comment]
Congrats Ylian!  I'm a long time Microsoft Partner (although not a software developer) so good to see you move to another company that is crucial to the success of my IT services/VAR/MSP business.   Hopefully they will encourage/allow you to continue the MeshCentral project. If anything maybe it will allow for greater integration with systems running Microsoft's various OS's.

**Comment by frakman1** [comment]
I was going to write that I was sorry to hear of your job loss, especially after being with them for so many years (I've been with the same employer for 23 years), but by the time I wrote it, I see that Microsoft was lucky enough to have hired you. So ...

<div id="user-content-toc">
  <ul>
    <summary><h1 style="display: inline-block;">🎉 Congratulations! You are a gift to the software development community</h1></summary>
  </ul>
</div>

My thoughts on monetizing Mesh Central is to offer management of 20 devices for free and require payment for anything above that. Also limit the number of groups. Sort of how Tailscale works which I view as an adjacent software service (I know they're very different but they achieve the same goal of remote management). Ngrok has a similar model too. You don't have to be an all-out business owner as others commented. It can be a side-gig with minimal management through software and a steady income stream from subscriptions.

I think that would be a good balance between users who just want to manage a few personal devices for free and corporations with bigger inventories.

**Comment by si458** [comment]
follow on post i discovered for anybody intrested
https://meshcentral2.blogspot.com/2023/02/starting-work-at-microsoft.html

**Comment by renueman** [comment]
I started using Meshcentral about a year ago to admin a LOT of remote systems. I am worried that its not supported anymore because of the employment change. Is this considered a dead project now or is someone taking over the reins?

**Reply by si458** [reply]
@Ylianst is still accepting PRs and stuff in his free time and nobody has really taken over the project,
i was thinking over forking and trying to keep it up to date with PRs from people myself
but my experience is only nodejs, i couldnt fix issues with the agent for example as thats C

**Reply by renueman** [reply]
Perhaps Open-sourcing the project would be an option? I came from using teamviewer and had to pay their blood money yearly for its use even though I had originally bought the lifetime subscription type deal. So, I was very excited to find MC and did quite a bit of work setting up all my systems to use it. It would be pretty depressing to find out it was all for naught. I did have a conversation with Ylianst about paying for it in the form of donations but he refused. MC is such great software that I sure do not want to see it die off.

**Reply by dinger1986** [reply]
The project is already opensource, it is secure and stable so isnt dead.

**Reply by si458** [reply]
it already is sorta opensource, check the LICENSE - Apache

**Reply by renueman** [reply]
I didnt realize that - sorry. That is certainly some good news really. What would be nice is an actual forum setup for MC. A place where everyone using it can discuss it. I have always looked at Github just for reporting bugs.

**Reply by marclaporte** [reply]
I don't understand why you would say "sorta opensource".

The Apache license is unequivocally Open Source: https://opensource.org/license/apache-2-0/

**Reply by cfoellmann** [reply]
like @marclaporte says, Apache 2.0 is at the forefront of opensource.
meshcentral is already in distribution with https://github.com/amidaware/tacticalrmm . The devs are committed to keep their packaged version fixed and ready for deployment.

**Reply by marclaporte** [reply]
ok, but "The Tactical RMM License is not an open-source software license." Source: https://github.com/amidaware/tacticalrmm/blob/develop/LICENSE.md

**Reply by cfoellmann** [reply]
Yes it is not a proper opensource license.
We were talking about the meshcentral license. since that is apache-2.0 it allows for packaging with other software as long as the changes to meshcentral are made public and open. That way fixes from the trmm team will be available to the meshcentral project. For them it would be even better if meshcentral would be continued and they could contribute "upstream" so they can use a working copy without much maintainance on their side.

**Reply by marclaporte** [reply]
@cfoellmann AFAICT, you don't work for the Tactical RMM team and are speculating about their intentions. Can you share a URL about any official or unofficial intentions by the Tactical RMM team?

> since that is apache-2.0 it allows for packaging with other software as long as the changes to meshcentral are made public and open

Wikipedia says: "The Apache License is permissive; unlike copyleft licenses, it does not require a derivative work of the software, or modifications to the original, to be distributed using the same license." Source: https://en.wikipedia.org/wiki/Apache_License


> The devs are committed to keep their packaged version fixed and ready for deployment.
> For them it would be even better if meshcentral would be continued and they could contribute "upstream" so they can use a working copy without much maintainance on their side

That makes sense if upstream is maintained and healthy like it has been for many years. But on https://docs.tacticalrmm.com/mesh_integration/, it states:
1- "At some point in the future, these functions will be directly built into the Tactical agent, removing the need for MeshCentral."
and 2- "if it becomes necessary to fix something that breaks or packages need updating we are prepared to begin maintaining our own fork"

So another potential future is that the Tactical RMM team maintains a fork for a short while (potentially with a non Open Source license) while they develop a built-in replacement (their stated future plan), and abandon MeshCentral altogether. I am not predicting this, I am just saying it's a possible scenario based on official information from the Tactical RMM team.

**Reply by cfoellmann** [reply]
@marclaporte you are totally right. I was wrong apache is that open that it is not copyleft.
I can not speak for the tacticalrmm project and I have not had any contact in the past. From a developer standpoint a fork is a lot more work than committing fixes upstream and use the release version in you product. I was looking at the problems in the usage of meshcentral within tacticalrmm and I was suprised that the trmm team have not send some extension to the meshcentral project to make it simple and better for them to integrate the functionality.

I am happy that you are replying and we are having some kind of (limited) conversation. The lack of traffic on this topic is giving me a bad feeling about the future of meshcentral.

**Reply by wh1te909** [reply]
it's not dead and @Ylianst is still accepting PR's and doing releases and mesh is a solid product that works extremely well so not sure why everyone is so worried. If it truly does become abandoned then yes we are prepared to maintain a fork.

**Comment by lfabry** [comment]
Hi there



I am definitely keen to help to make sure Meshcentral can be maintained and extended for the long run.
It is definitely a good idea to have a place to discuss how to move forward.
MeshCentral is quite a stable platform but some feature requests are starting to pileup on my side.

What about building a core team to move forward?

Laurent

**Comment by marclaporte** [comment]
Relevant: https://meshcentral2.blogspot.com/2023/10/meshcentral-windows-arm64-nodejs-v11.html

I urge everyone to join me in sponsoring Simon Smith (si458)!
https://github.com/sponsors/si458

We need your contributions: time, code or money!
https://github.com/Ylianst/MeshCentral/discussions/5540

**Comment by Magneticdud** [comment]
I'm sorry to discover this.

Unfortunately, intel in Q3 22 *only* had $15 billion in revenue, down 20% from the past year, so *clearly* it's the best engineers who really need to go first.

Please think to the poor shareholders that earned only 25 cents per share. That's *only* $95 million for Vanguard and *just* $80 million for Blackrock. I'm literally crying for them.

Who cares if the official tool (Intel Manageability Commander) is based on an ancient 32-bit version of electron (it's totally safe for a remote control solution with admin rights, not a problem having 20+ CVE), customers are dumb, they will still buy intel stuff.

**Reply by vithurshanselvarajah** [reply]
Without making this political. I think part of the reason why intels profits have been dropping is linked to their competitors upping their game. Handhelds now use Ryzen U chips, PS5 and 4 is AMD, avx512 is openly supported on Ryzen for those who want emulation. 3d cache is AMD. iGPU is best with AMD. Best we got from intel was E and P cores which isn't very attractive to consumers as it's confusing for those who aren't tech savvy and even then the scheduler that came with it isn't the best! Last intel chip I had was a i5 12th gen which I quickly replaced with AMD. Even my servers are ARM now due to their superior efficiency! 

I think it's a case of Intel got comfortable. Their new server chips draw a lot of power for not much in terms of benefit whilst EPYC albeit does do high draw actually provides a insane number of cores, PCIE lanes etc within the same package. Plus upfront cost is lower then intel counterpart in a lot of circumstances. 

Intel has some serious catching up to do before I'd consider buying more of their products.

**Comment by marclaporte** [comment]
"In November 2022, Intel discontinued its investment in, and is no longer contributing to, the Mesh open-source project." Source: https://www.intel.com/content/www/us/en/content-details/773384/intel-discontinues-contribution-to-mesh-open-source-project.html

The community has stepped up with:

- 31 releases between 2022-12 and 2024-10: https://github.com/Ylianst/MeshCentral/releases
- A major revamp of the UI: https://github.com/Ylianst/MeshCentral/pull/6450
- And we are starting monthly community meetings: https://github.com/Ylianst/MeshCentral/wiki/Community-Monthly-Meetings

Congrats to all, especially @si458 and @Ylianst !

**Comment by PathfinderNetworks** [comment]
I'm not a coder nor software developer but have been an active MeshCentral user going back at least 10 years with version 1.  Can I take part in the monthly meetings?  I would mostly represent the user/MSP/IT Manager side of things.

**Reply by si458** [reply]
Sure, come along to the meeting!
anybody is welcome to join! 
Happy to meet new people and hear views on things :)

**Comment by lfabry** [comment]
I ll do my best to join the meeting.

**Comment by PathfinderNetworks** [comment]
I'm guessing the first community meeting must not be today (Oct 24th)?  I joined the meeting at 10am EDT and there were no other participants. Checked again about 5-6 minutes later just in case.  What is the date of the first meeting?  I know they are to be held the 4th Thursday of the month.

**Reply by si458** [reply]
yes sorry this was a mistake on our part!
we meant to say the first meeting will be on the 31st Oct then afterwards every 4th thursday!
so sorry again about this!
